doc/Methods2.Rmd

```diff
@@ -0,0 +1,11 @@
+---
+title: "Data Analysis"
+author: "Farida Samad-zada"
+date: '33.12.2017 '
+output: html_document
+---
+
+##Methods
+###Data Analysis 
+Despite the fact that the original dataset contained 17 orders, we narrowed down the dataset, by including only the orders, where number of observations was more than 30.  Hence, only Rodentia, Cetacea, Insectivora, Carnivora, Lagomorpha, Primates, Artiodactyla were used in the analysis.  Additionally, we log transformed our data in order to be able to implement linear regression into our analysis.
+All the modeling and statistical analyses were conducted using R version 3.4.1 (R Core Team  2017).  We analyzed the number of offspring per year, as well as the maximum life span, weaning age and the age of first reproduction using generalized linear model and mixed effects model . Glm and lme4 packages for used for modeling.  Log transformed values of weaning age, maximum lifespan and age of first reproduction were used as fixed effects. Additionally, in the mixed effects model, orders were used as a random effect factor in order to account for life-history variations, as well as size differences between orders.  We compared the two models using Akaike Information Criterion (AIC) to see if additional parameters significantly improved the fit of the model. Since  mixed effects model was a better fit than the generalized linear model, in our results and discussion sections, we only focus on the former. 
```

Comment:
It's a good idea to hyperlink `glm` and `lme4` to their respective CRAN sites the first time you mention them.